net/node/transactionPool.go

```diff
@@ -64,3 +92,15 @@ func (node *node) SynchronizeTxnPool() {
 		}
 	}
 }
+
+func (txnPool *TXNPool) CleanSubmittedTransactions(block *ledger.Block) error {
+	txnPool.Lock()
+	defer txnPool.Unlock()
+	log.Trace()
+
+	err := txnPool.CleanTxnPool(block.Transactions)
```

Comment:
thanks for your review indeed , I have remove the value copy,  and transfer the transaction array to next func.